cmake/tests.cmake

```diff
@@ -214,4 +214,5 @@ target_link_libraries(lite-arena-test libprotobuf-lite gmock_main)
 
 add_custom_target(check
   COMMAND tests
+  DEPENDS tests test_plugin
```

Comment:
Ah ok, thanks. I'll just leave in the `tests` dependency then; if it turns out to be unnecessary then it won't cause any harm anyway.